koku/masu/processor/_tasks/process.py

```diff
@@ -76,22 +75,21 @@ def _process_report_file(schema_name, provider, provider_uuid, report_dict):
         manifest_id=manifest_id,
     )
 
-    with transaction.atomic():
-        processor.process()
-        with ReportStatsDBAccessor(file_name, manifest_id) as stats_recorder:
-            stats_recorder.log_last_completed_datetime()
+    processor.process()
+    with ReportStatsDBAccessor(file_name, manifest_id) as stats_recorder:
+        stats_recorder.log_last_completed_datetime()
 
-        with ReportManifestDBAccessor() as manifest_accesor:
-            manifest = manifest_accesor.get_manifest_by_id(manifest_id)
-            if manifest:
-                manifest.num_processed_files += 1
-                manifest.save()
-                manifest_accesor.mark_manifest_as_updated(manifest)
-            else:
-                LOG.error("Unable to find manifest for ID: %s, file %s", manifest_id, file_name)
+    with ReportManifestDBAccessor() as manifest_accesor:
+        manifest = manifest_accesor.get_manifest_by_id(manifest_id)
+        if manifest:
+            manifest.num_processed_files += 1
```

Comment:
Since you're here, I think you can knock out #2118 while you're at it 😄 

Instead of incrementing, I'm pretty sure we just need to actually count the number of files that have been processed, like this:
```suggestion
            manifest.num_processed_files = CostUsageReportStatus.objects.filter(manifest_id=manifest_id, last_completed_datetime__isnull=False).count()
```